src/openforms/js/components/admin/form_design/registrations/objectsapi/LegacyConfigFields.js

```diff
@@ -183,7 +183,10 @@ const LegacyConfigFields = ({index, name, schema, formData, onFieldChange}) => {
           id="root_uploadSubmissionCsv"
           name="uploadSubmissionCsv"
           value={uploadSubmissionCsv}
-          onChange={onFieldChange}
+          checked={uploadSubmissionCsv}
```

Comment:
I'd expect React warnings in the console here because an `input` element can only get string values :thinking:

Reply:
I don't get any react warnings for this, but I do get though a lot for `objecttypeVersion` :

_Invalid prop `formData.objecttypeVersion` of type `number` supplied to `LegacyConfigFields`, expected `string`_

Reply:
yeah, let's ignore those for now :(